__tests__/apps.test.js

```diff
@@ -64,6 +63,25 @@ describe("app", () => {
         
       
     })
+    describe("GET /api/articles/:article_id", () => {
+        test('200: GET - an articles array of article objects, each of which should have the following properties: author, title, article_id, topic, created_at, votes, article_img_url, comment_count', () => {
+            return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.article).toEqual({
+                    article_id: 3,
```

Comment:
Rather than using 'toEqual' I would recommend using the matchers either objectContaining or toMatchObject in order to make the testing more flexible, in case we add any extra collums later on- this means you won't need to go back and change this test to pass.